services/ui-src/src/components/cards/EntityCard/QualityMeasuresSection.tsx

```diff
@@ -117,22 +157,28 @@ export const QualityMeasuresSection = ({
   topSection,
   bottomSection,
 }: Props) => {
+  // LaunchDarkly
+  const newQualityMeasuresSectionEnabled =
+    useFlags()?.newQualityMeasuresSectionEnabled;
```

Comment:
this is reading off the flag, but we will have a point in the future where the flag is enabled (or gone) and we have reports with both the old qm and the new qm sections. Users may still view the report or the PDF for these older reports.